test/gtfs_test.exs

```diff
@@ -47,7 +39,13 @@ defmodule GtfsTest do
           ]
         })
 
-      refute Enum.member?(Gtfs.all_routes(pid), %Route{id: "Red", directions: %{}})
+      refute Enum.member?(Gtfs.all_routes(pid), %Route{
+               id: "Red",
+               direction_names: %{
+                 0 => "Outbound",
+                 1 => "Inbound"
```

Comment:
`Red`'s direction names would be `North` and `South`. Could you refute that the id `Red` exists anywhere rather than trying to make a struct that would exactly match?